lib/shared/types/src/types/config/models/variable.ts

```diff
@@ -40,7 +40,8 @@ export enum VariableType {
  * Supported variable values
  */
 export type DVCJSON = { [key: string]: string | boolean | number }
-export type VariableValue = string | boolean | number | DVCJSON
+export type VariableValue = string | boolean | number | Record<string, unknown> |
```

Comment:
are these used by the JS or Node SDKs at all? If so I'd probably separate the type for those so that it's not possible to use arrays on those SDKs as default values

Reply:
I'm really not sure how thats going to work without completely overhauling all of the types. Ultimately the defaultValue type has to be same as the variable value type, and I dont see how I can decouple those from the variable value types in the bucketing config